### ISSUE TITLE
Rotated fire 

### ISSUE THREAD

**Issue by BenkinsMcParbo** [toggledbits/MatrixFireFast#3]
Hello!

Apologies for what’s probably a super basic question but we’re trying to do this with a 32x8 matrix and the fire seems to be being generated from the short side instead of one of the long ones.
It’s like the image is rotated 90 degrees.
We’ve tried changing instances of “row” in the code to “column.” We’ve tried defining or undefining parts to change the origin but it seems to only flip it from one end to the other.
We think we’re missing something really obvious so if you’ve got any tips that would be amazing.

Cheers,
Benkins

**Comment by toggledbits** [comment]
If defining or undefining the `PANEL_TOP` and `PANEL_ZIGZAG` constants don't get you what you need, you can write your own `pos()` function replacement, which takes a row and column index and converts it into a pixel address (a single number that addresses the right LED in the string). That is [described here](/toggledbits/MatrixFireFast#customizing-the-pos-function).

Before you go that route, have you tried defining `MAT_COL_MAJOR`?

**Comment by BenkinsMcParbo** [comment]
Hi! Thanks for the response, and apologies for not getting back to you. We did fix it in the end - I believe we changed the instances of "rows" to "columns" and that seemed to force the solution. We were also using a slightly different sized matrix, and we changed the number of pixels in the code, and we also did the things you suggested above. But for some reason it was the column and row thing that seemed to do the trick. Thanks again, the code looked amazing.
